lib/sites/cvs.rb

```diff
@@ -0,0 +1,202 @@
+require 'date'
+require 'json'
+require 'rest-client'
+
+require_relative './base_clinic'
+
+module Cvs
+  STATE = 'MA'
+  USER_AGENTS = []
+  CVS_CITIES = {}
+
+  class CvsCity
+    attr_reader :city, :stores, :should_tweet
+
+    def initialize(city, stores, should_tweet)
+      @city = city
+      @stores = stores
+      @should_tweet = should_tweet
+    end
+  end
+
+  File.open("user_agents.txt", "r") do |f|
+    f.each_line do |line|
+      USER_AGENTS.append(line.strip)
+    end
+  end
+
+  File.open("cvs_locations.csv", "r") do |f|
+    f.each_line do |line|
+      entry = line.strip.split(',')
+      city = entry[0]
+      stores = entry[1].to_i
+      should_tweet = entry[2] == "1"
+      CVS_CITIES[city] = CvsCity.new(city, stores, should_tweet)
+    end
+  end
+
+  # This array also defines the order in which city names appear, the order is determined
+  # by the number of stores in a city.
+  TWEET_ALLOWED_CITIES = CVS_CITIES.keys.filter { |cvs_city| CVS_CITIES[cvs_city].should_tweet }
+                                   .sort_by { |cvs_city| -1 * CVS_CITIES[cvs_city].stores }
+
+  def self.state_clinic_representation(storage, logger)
```

Comment:
We should wrap this whole method in a `SentryHelper.catch_errors` block so that it can't crash the other site scrapers. That method is nice because it will still raise errors in local environments for easier debugging.